Azure.PowerShell.Netcore.sln

```diff
@@ -111,8 +111,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Commands.Reservations.Netco
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Commands.Scheduler.Netcore", "src\ResourceManager\Scheduler\Commands.Scheduler\Commands.Scheduler.Netcore.csproj", "{A65322AC-2B28-482C-84A5-59D8BAF2F46D}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Commands.ServerManagement.Netcore", "src\ResourceManager\ServerManagement\Commands.ServerManagement\Commands.ServerManagement.Netcore.csproj", "{A6D19586-2219-440C-A7C9-EA729DB76294}"
```

Comment:
@maddieclayton make sure to remove the [project GUID references down below](https://github.com/Azure/azure-powershell/blob/preview/Azure.PowerShell.Netcore.sln#L826-L837)